packages/wpcom.js/examples/browser-cors/package.json

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I'd prefer to drop the package files for examples; there's not much point in them.